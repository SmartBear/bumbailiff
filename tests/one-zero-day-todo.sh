mkdir -p tmp/git/one-zero-day-todo
cd tmp/git/one-zero-day-todo
git init --quiet .
echo "// TODO: 0-day old TODO" > file.js
git commit --quiet --allow-empty -m "Commit"
bumbailiff
echo $?
