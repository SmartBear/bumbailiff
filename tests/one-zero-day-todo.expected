mkdir -p tmp/git/one-zero-day-todo
cd tmp/git/one-zero-day-todo
git init --quiet .
echo "// TODO: 0-day old TODO" > file.js
git commit --quiet --allow-empty -m "Commit"
bumbailiff
[1m[0m
ðŸŒ± [0;32m0 TODO-days accumulated (max allowance: 14 ðŸ’µ).[0m
echo $?
0
